packages/server/tests/integration/v1/boards.spec.ts

```diff
@@ -108,20 +127,149 @@ describe("POST /api/v1/boards", () => {
     expect(response.body.code).toBe("NOT_ENOUGH_PERMISSION");
   });
 
-  // it("should create a board", async () => {
-  //   // seed users with "board:create permission"
-  //   const createUser = await database
-  //     .insert([
-  //       {
-  //         userId: uuid(),
-  //         email: "create-board@example.com",
-  //         password: hashPassword("strongPassword"),
-  //         username: "create-board",
-  //       },
-  //     ])
-  //     .into("users")
-  //     .returning(["userId"]);
-  // });
+  it("should create a board", async () => {
+    // seed users with "board:create permission"
+    // const board : BoardInsertRecord = generateBoards({}, false);
+    const { user: authUser } = await createUser();
+
+    await createRoleWithPermissions(authUser.userId, ["board:create"], {
+      roleName: "Board Creator",
+    });
+
+    const response = await supertest(app)
+      .post(`/api/v1/boards/`)
+      .set("Authorization", `Bearer ${authUser.authToken}`);
+    // .send({
+    //   boardId: board.boardId,
+    //   name: board.name,
+    //   url: board.url,
+    //   color: board.color,
+    //   display: board.display
+    // });
+  });
+});
+
+describe("PATCH /api/v1/boards", () => {
+  it('should throw error "INVALID_AUTH_HEADER"', async () => {
+    const response = await supertest(app).patch("/api/v1/boards");
+
+    expect(response.headers["content-type"]).toContain("application/json");
+    expect(response.status).toBe(400);
+    expect(response.body.code).toBe("INVALID_AUTH_HEADER");
+  });
+
+  it("should throw error 'INVALID_AUTH_HEADER_FORMAT'", async () => {
+    const { user: authUser } = await createUser();
+
+    const response = await supertest(app)
+      .patch("/api/v1/boards")
+      .set("Authorization", `Beare${authUser.authToken}`);
+
+    expect(response.headers["content-type"]).toContain("application/json");
+    expect(response.status).toBe(401);
+    expect(response.body.code).toBe("INVALID_AUTH_HEADER_FORMAT");
+  });
+
+  it("should throw error 'BOARD_NOT_FOUND'", async () => {
+    const board: BoardInsertRecord = await generateBoards({}, false);
+    const { user: authUser } = await createUser();
+
+    await createRoleWithPermissions(authUser.userId, ["board:update"], {
+      roleName: "Board Patcher",
+    });
+
+    const response = await supertest(app)
+      .patch("/api/v1/boards")
+      .set("Authorization", `Bearer ${authUser.authToken}`)
+      .send({
+        boardId: board.boardId,
```

Comment:
Instead of seeding a board in this test case.

You can simply use a random generated ID